docs/Updating-npm-bundled-node-gyp.md

```diff
@@ -41,6 +41,13 @@ $ npm explore npm/node_modules/npm-lifecycle -g -- npm install node-gyp@latest
 
 If the command fails with a permissions error, please try `sudo` and then the command.
 
+If you are using `nvm` and the logs indicate no change of the `node-gyp` version, you might need to specify the `node` version, for example do:
+```bash
+$ cd /home/user/.nvm/versions/node/v18.13.0/node_modules/npm/node_modules/@npmcli/run-script
```

Comment:
Thanks for doing this.  Please clarify for the reader how you chose this path.  Also, what about adding `npm list` before and after so the reader knows how to verify that the expected changes were made.

Reply:
While you are in here, can you please update line 32 to say `7 or greater` you cover npm 9.